app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -654,6 +654,34 @@ impl NodeModel {
             self.display_object.add_child(&self.error_indicator);
         }
     }
+
+    /// Return the `SinglePortView` of the first output port of the node.
+    ///
+    /// Returns `None`:
+    /// 1. If there are no output ports.
+    /// 2. If the port does not have a `PortShapeView`.
```

Comment:
When the node has no PortShapeView? Try to figure out, but don't spend much time on it.